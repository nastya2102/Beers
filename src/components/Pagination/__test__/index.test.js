import React from 'react'
import { render } from '@testing-library/react'
import Pagination from '../index'

describe('Pagination', () => {
  test('should render Pagination component', () => {
    const { container } = render(<Pagination active={1} onChange={() => {}} isLastPage={true}/>)

    expect(container).toBeTruthy()
  })

  test('should match snapshot Pagination component', () => {
    const { container } = render(<Pagination active={1} onChange={() => {}} isLastPage={true}/>)

    expect(container).toMatchSnapshot()
  })
})
