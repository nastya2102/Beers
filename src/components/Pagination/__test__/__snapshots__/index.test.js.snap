// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination should match snapshot Pagination component 1`] = `
<div>
  <div
    class="sc-bdfBwQ bEhSYe"
  >
    <span
      class="sc-gsTCUz gkrfUS"
      disabled=""
    >
      «
    </span>
    <span
      class="sc-gsTCUz dlHzSm"
    >
      -3
    </span>
    <span
      class="sc-gsTCUz dlHzSm"
    >
      -2
    </span>
    <span
      class="sc-gsTCUz dlHzSm"
    >
      -1
    </span>
    <span
      class="sc-gsTCUz dlHzSm"
    >
      0
    </span>
    <span
      class="sc-gsTCUz bKyJRQ"
    >
      1
    </span>
    <span
      class="sc-gsTCUz gkrfUS"
      disabled=""
    >
      »
    </span>
  </div>
</div>
`;
